paprika/restraints/openmm.py

```diff
@@ -2,8 +2,14 @@
 import logging
 
 import numpy as np
-import openmm as openmm
-import openmm.unit as openmm_unit
+
+try:
+    import simtk.openmm as openmm
+    import simtk.unit as openmm_unit
+except ImportError:
+    import openmm
+    import openmm.unit as openmm_unit
+
```

Comment:
Maybe flip this and try the new style import first?

Reply:
Thank you for the tip, this is a good point.